android/pytorch_android/build.gradle

```diff
@@ -79,4 +79,49 @@ task sourcesJar(type: Jar) {
     classifier = 'sources'
 }
 
+def getLibtorchHeadersDir() {
+  def abi = ABI_FILTERS.split(",")[0]
+  return "$rootDir/../build_android_$abi/include";
+}
+
+afterEvaluate {
+  if (POM_PACKAGING == 'aar') {
+    android.libraryVariants.all { variant ->
+      variant.outputs.each { output ->
+        File f = output.outputFile
+        if (f.name.endsWith(".aar")) {
```

Comment:
I'm not familiar with `android.libraryVariants.all` above - is there supposedly only one AAR file?

Reply:
for `pytorch_android` variants.all will include `debug` and `release`.